backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/CassandraCluster.java

```diff
@@ -33,13 +35,23 @@
 public final class CassandraCluster implements AutoCloseable {
     public static final String KEYSPACE = "testing";
 
+    private static Optional<Exception> startStackTrace = Optional.empty();
     private final CassandraModule module;
     private Session session;
     private CassandraTypesProvider typesProvider;
     private Cluster cluster;
 
     public static CassandraCluster create(CassandraModule module, Host host) {
-        return new CassandraCluster(module, host);
+        assertClusterNotRunning();
+        CassandraCluster cassandraCluster = new CassandraCluster(module, host);
+        startStackTrace = Optional.of(new Exception());
```

Comment:
can we add a description to this exception to explain that it is the root cause, meaning where the first cluster was launched?